plugin.php

```diff
@@ -306,7 +306,7 @@ function rest_validate_request_arg( $value, $request, $param ) {
 		if ( isset( $args['format'] ) ) {
 			switch ( $args['format'] ) {
 				case 'date-time' :
-					if ( ! rest_parse_date( $value ) ) {
+					if ( $value && ! rest_parse_date( $value ) ) {
```

Comment:
> Also, would this break things if we handled it that way?

Potentially. Someone should look at existing validation libraries to see how null cases are handled.